lib/src/editor/ui/editor.dart

```diff
@@ -61,17 +61,20 @@ class Editor extends StatelessWidget {
                     ),
                     Consumer<ZoomProvider>(
                       builder: (context, model, child) {
-                        return context.watch<DevicePreviewProvider>().show ? DevicePreview(
-                          builder: (context) {
-                            return Transform.scale(
-                              scale: model.zoom,
-                              child: component ?? const SizedBox.shrink(),
-                            );
-                          },
-                        ) : Transform.scale(
-                              scale: model.zoom,
-                              child: component ?? const SizedBox.shrink(),
-                            );
+                        return context.watch<DevicePreviewProvider>().show
+                            ? DevicePreview(
+                                builder: (context) {
+                                  return InteractiveViewer(
```

Comment:
The goal was to add pan and zoom, at large, not just within `DevicePreview`

Reply:
I was looking at the storybook example and I noticed they don't have panning, they only have transformation scale. If we wanted to ad interactive gesture to the default, I think we would either have to remove the the minus and plus button for scale and just use the Interactive Viewer.

https://next--storybookjs.netlify.app/official-storybook/?path=/story/addons-controls--custom-control-matchers

Reply:
@devbrandonlong